content/docs/portals.md

```diff
@@ -65,10 +65,11 @@ This includes event bubbling. An event fired from inside a portal will propagate
 </html>
 ```
 
-A `Parent` component in `#app-root` would be able to catch an uncaught, bubbling event from the sibling node `#modal-root`.
+`#app-root` में एक `पैरेंट` कौम्पोनॅन्ट अपने सिबलिंग नोड `#modal-root` से एक अज्ञात बबलिंग इवेंट को पकड़ने में सक्षम होगा।
 
-```js{28-31,42-49,53,61-63,70-71,74}
-// These two containers are siblings in the DOM
+
```

Comment:
why the change in numbers?

Reply:
did this because after translating the comments number of lines were reduced by 2